tests/kamereon/test_kamereon_vehicle_data.py

```diff
@@ -329,6 +328,36 @@ def test_location_v2() -> None:
     assert vehicle_data.gpsLongitude == 11.1234567
     assert vehicle_data.lastUpdateTime == "2020-02-18T16:58:38Z"
 
+def test_lock_status() -> None:
```

Comment:
Need two carriagr returns between functions here
```suggestion

def test_lock_status() -> None:
```